framework/xds_k8s_testcase.py

```diff
@@ -682,6 +682,84 @@ def assertAllBackendsReceivedRpcs(self, lb_stats):
                 msg=f"Backend {backend} did not receive a single RPC",
             )
 
+    def assertClientEventuallyReachesSteadyState(
+        self,
+        test_client: XdsTestClient,
+        rpc_type: str,
+        num_rpcs: int,
+        threshold_percent: int,
+        *,
+        retry_timeout: dt.timedelta = dt.timedelta(minutes=20),
+        retry_wait: dt.timedelta = dt.timedelta(seconds=2),
+        steady_state_delay: dt.timedelta = dt.timedelta(seconds=5),
+    ):
+        retryer = retryers.constant_retryer(
+            wait_fixed=retry_wait,
+            timeout=retry_timeout,
+            error_note=(
+                f"Timeout waiting for test client ${test_client.hostname} to"
+                f"report {num_rpcs} pending calls +/-{threshold_percent}%"
+            ),
+        )
+        for attempt in retryer:
+            with attempt:
+                self._checkRpcsInFlight(
+                    test_client, rpc_type, num_rpcs, threshold_percent
+                )
+        time.sleep(steady_state_delay.total_seconds())
+        self._checkRpcsInFlight(
+            test_client, rpc_type, num_rpcs, threshold_percent
```

Comment:
Why do we need check this again after retries?

Reply:
The checks in the retries validate that the number of RPCs reaches the target number. This check validates that the number of RPCs is steady over a period of time.